src/digisos/redux/ettersendelse/ettersendelseSaga.ts

```diff
@@ -141,20 +141,26 @@ function* lastOppEttersendelsesVedleggSaga(
         }
         let feilKode: REST_FEIL = detekterInternFeilKode(reason.toString());
         // Kjør feilet kall på nytt for å få tilgang til feilmelding i JSON data:
+        //@ts-ignore
         response = yield call(fetchUploadIgnoreErrors, url, formData);
         const ID = "id";
         // @ts-ignore
         if (response && response[ID]) {
             // @ts-ignore
             feilKode = response[ID];
         }
-        yield put(lastOppEttersendelseFeilet(feilKode, opplysningType.toString()));
+        yield put(
+            lastOppEttersendelseFeilet(feilKode, opplysningType.toString())
+        );
         if (
             feilKode !== REST_FEIL.KRYPTERT_FIL &&
             feilKode !== REST_FEIL.SIGNERT_FIL
         ) {
             yield put(
-                loggInfo("Last opp vedlegg for ettersendelse feilet: " + reason.toString())
+                loggInfo(
+                    "Last opp vedlegg for ettersendelse feilet: " +
```

Comment:
Kan vi ha disse på en linje?

Reply:
Jeg økte maksbredden for linjer i prettierconfigen slik vi har gjort i modia. Håper at 120chars bredde er tilstrekkelig i de fleste tilfeller. :)